repo/packages/Q/quobyte/0/config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "object",
+  "properties": {
+    "quobyte": {
+      "type": "object",
+      "description": "Quobyte Framework specific configuration properties",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "quobyte"
+        },
+        "zk": {
+          "type": "string",
+          "description": "Name or IP address of the zookeeper server.",
+          "default": "$ZOOKEEPER"
+          },
+        "master": {
+          "type": "string",
+          "description": "Name, IP address or Zookeeper URL of the Mesos master server.",
+          "default": "$MESOS_MASTER"
```

Comment:
this is an unusual configuration.  I'm expecting something like: https://github.com/mesosphere/multiverse/blob/version-1.x/repo/packages/S/swarm/0/config.json#L4-L11